bundle/build.rs

```diff
@@ -0,0 +1,130 @@
+use cid::multihash::{Code, MultihashDigest};
+use cid::Cid;
+use fil_actor_bundler::Bundler;
+use std::error::Error;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::process::{Command, Stdio};
+use std::thread;
+
+/// Cargo package for an actor.
+type Package = str;
+
+/// Technical identifier for the actor in legacy CodeCIDs and else.
+type ID = str;
+
+const ACTORS: &[(&'static Package, &'static ID)] = &[
+    ("system", "system"),
+    ("init", "init"),
+    ("cron", "cron"),
+    ("account", "account"),
+    ("multisig", "multisig"),
+    ("power", "storagepower"),
+    ("miner", "storageminer"),
+    ("market", "storagemarket"),
+    ("paych", "paymentchannel"),
+    ("reward", "reward"),
+    ("verifreg", "verifiedregistry"),
+];
+
+const IPLD_RAW: u64 = 0x55;
+const FORCED_CID_PREFIX: &str = "fil/6/";
+
+fn main() -> Result<(), Box<dyn Error>> {
+    // Cargo executable location.
+    let cargo = std::env::var_os("CARGO").expect("no CARGO env var");
+    println!("cargo:warning=cargo: {:?}", &cargo);
+
+    let out_dir = std::env::var_os("OUT_DIR").expect("no OUT_DIR env var");
+    println!("cargo:warning=out_dir: {:?}", &out_dir);
```

Comment:
Remove these warnings?

Reply:
I actually want to keep them as feedback and for debugging purposes. Using `cargo:warning` is a hack, but the alternative would be to hijack the process' stdout handle, and we weren't able to find a portable way of doing that across OSes.